source/keymap.js

```diff
@@ -12,7 +12,8 @@
 			19  : 'pause',
 			412 : 'rewind',
 			417 : 'fastforward',
-			461 : 'back'
+			461 : 'back',
+			8	: 'back'	//webkit keycode of backspace key in keyboard
```

Comment:
I'm wondering if we shouldn't add this new mapping in a more general location (as keyCode 8 for "backspace" is native to JavaScript and we probably want this behavior to occur on PC as well); these mappings here are conditionalized for webOS devices. Or alternatively, just handling keyCode 8 directly in `moon.History`, similar to how we are handling keyCode 13 for "enter" throughout the framework in general.